routes/web.php

```diff
@@ -67,19 +67,25 @@
 
     // Printables
 
-    // TODO : print feels... unRESTY; revise
-    Route::get('/registration/{registration}/print', [
+    // Print a specific Family Form for User Centre (Edit page)
+    Route::get('/registrations/{registration}/print', [
         'as' => 'service.registration.print',
-        'uses' => 'Service\RegistrationController@print',
+        'uses' => 'Service\RegistrationController@printOneIndividualFamilyForm',
     ]);
 
-    // TODO : print feels... unRESTY; revise
-    Route::get('/centre/{centre}/registrations/print', [
-        'as' => 'service.centre.registrations.print',
-        'uses' => 'Service\CentreController@printRegistrations',
+    // Batch print Family Forms for User Centre
+    Route::get('/registrations/print', [
+        'as' => 'service.registrations.print',
+        'uses' => 'Service\RegistrationController@printBatchIndividualFamilyForms',
     ]);
 
-    // Exportable reports
+    // Print a Specific Centre's Registration's register form
+    Route::get('/centre/{centre}/registrations/register-form', [
```

Comment:
Still a bit confused about what all of these are - collection vs individual - but I think this might be the collection sheet.